### ISSUE TITLE
LaTeX labels

### ISSUE THREAD

**Issue by alexpearce** [alexpearce/histograms#6]
Axes labels should be rendered as LaTeX.

Using [MathJax](http://www.mathjax.org/) this might not be too hard.
The problem is that ROOT uses [`TLatex`](http://root.cern.ch/root/html534/TLatex), some odd LaTeX look-a-like that isn't directly compatible.

For example, the `TLatex` string `Mass [MeV/c^{2}]` is not valid LaTeX, as the superscript `^` can only appear in math mode, and so the LaTeX string should be `Mass [MeV/$c^{2}$]`. (The `c` should be italicised too, which would require extra `TLatex` markup.)
